### DISCUSSION TITLE
Disable "Shorten URL" if user selects Expires under 1 year

### DISCUSSION THREAD

**Discussion by haywardgg** [PrivateBin/PrivateBin#1276] [General]
Does anyone know if this is possible ? 
If the user selects anything less than 1 year for Expire option, I don't want them to get the shortenURL option. 
I use Yourls for the URL shortner. 

Thanks

**Comment by rugk** [comment]
This would likely need a customisation of the JS/new JS, to be added to the template.

It's quite of a special wish and not covered/implemented.

**Comment by rugk** [comment]
Alternatively, you may just host two instances of PrivateBin. One having url shortener disabled and having only expire options less than a year and one having url shortener enabled and having (only?) an expire option of one year or longer.

This can all be configured, see https://github.com/PrivateBin/PrivateBin/wiki/Configuration for more details.
